docs/install/scala_setup.md

```diff
@@ -0,0 +1,87 @@
+# Setup the MXNet Package for Scala
+
+The following instructions are provided for macOS and Ubuntu. Windows is not yet available.
+
+**Note:** If you use IntelliJ or a similar IDE, you may want to follow the [MXNet-Scala on IntelliJ tutorial](../tutorials/scala/mxnet_scala_on_intellij.html) instead of these instructions.
+
+## Setup Instructions
+**Step 1.** Download the MXNet source.
+
+```bash
+git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet
+cd mxnet
+```
+
+**Step 2.** Install dependencies:
+
+**macOS Steps**
+
+```bash
+brew update
+brew tap caskroom/versions
+brew cask install java8
+brew install maven
+brew install opencv@2
```

Comment:
brew install opencv

Reply:
`opencv@2` is tested personally, and I know it works. I know that just `opencv` did not work.
Something to do with mxnet having already installed some version... brew wanted to install a later version, then had incompatibilities, etc, etc.

You also mention using v2 as a fix here: https://github.com/apache/incubator-mxnet/issues/11193#issuecomment-400460615